envoyextensions/extensioncommon/basic_envoy_extender.go

```diff
@@ -296,6 +296,27 @@ func (b *BasicEnvoyExtender) patchSupportedListenerFilterChains(config *RuntimeC
 func (b *BasicEnvoyExtender) patchListenerFilterChains(config *RuntimeConfig, l *envoy_listener_v3.Listener, nameOrSNI string) (*envoy_listener_v3.Listener, error) {
 	var resultErr error
 
+	// Special case for Permissive mTLS, which adds a filter chain
```

Comment:
I tried working with a sentinel name for the filter chain but ran into issues with v2 resource tests since v2 proxystate Routers don't have names.

Instead I added this special case code block to suppress errors as long as one filter chain gets patched successfully.